hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PreCheckException.java

```diff
@@ -82,9 +82,13 @@ public String toString() {
         return "PreCheckException{" + "responseCode=" + responseCode + '}';
     }
 
-    public static void validateTruePreCheck(boolean condition, ResponseCodeEnum errorStatus) throws PreCheckException {
-        if (!condition) {
+    public static void validateTruePreCheck(boolean flag, ResponseCodeEnum errorStatus) throws PreCheckException {
+        if (!flag) {
```

Comment:
I actually prefer `condition` here since more than just flags can be passed in

Reply:
Changed. Thanks